frontend/src/metabase-lib/expressions/helper-text-strings.ts

```diff
@@ -1,6 +1,19 @@
 import { t } from "ttag";
+import moment from "moment-timezone";
+import MetabaseSettings from "metabase/lib/settings";
```

Comment:
@metabase/frontend-developers, is there a recommended way to work through this linting rule?

Reply:
Ideally, split this into two PRs: the first one doing the move, and the second one adding the timezone info. It makes easy to isolate any linting issues (due to the first or the second) and to revert/understand the history.

Reply:
Done, thanks Ariya.